sound/soc/intel/common/soc-acpi-intel-hda-match.c

```diff
@@ -14,7 +14,8 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_hda_machines[] = {
 		/* .id is not used in this file */
 		.drv_name = "skl_hda_dsp_generic",
 		.sof_tplg_filename = "sof-hda-generic", /* the tplg suffix is added at run time */
-		.tplg_quirk_mask = SND_SOC_ACPI_TPLG_INTEL_DMIC_NUMBER,
+		.tplg_quirk_mask = SND_SOC_ACPI_TPLG_INTEL_DMIC_NUMBER |
+					SND_SOC_ACPI_TPLG_INTEL_BT_OFFLOAD,
```

Comment:
and this is going to break a lot of system presumably since we don not have any `-bt-ssp` topology out there..

Reply:
I removed all tplg name fixup code and let OSV to decide so we don't break all HDA PC...

Reply:
> I removed all tplg name fixup code and let OSV to decide so we don't break all HDA PC...

The problem is that we don't know who the OSV is, and we don't know how to distribute the required topology. So while the changes look ok at a high-level, who's going to use all this?

I don't want to merge something that creates a headache for support teams. We don't have a plan that makes any sense at the moment.

Reply:
Sorry I should provide the background story instead of just shooting the patches.

Currently there is a Chrome reference design brox which enables HDA external codec. Since BT audio offload was enabled for I2S Chromebooks, we want this feature could be also supported for HDA Chromebooks as well.

For the kernel driver, we will backport these two patches to Chrome tree and update the module option since Chromebook does not support NHLT table. And for topology, we will backport the patch to cavs2.5 branch and request Google to release the topology.